index.bs

```diff
@@ -1386,18 +1886,84 @@ partial interface MLGraphBuilder {
+            - *maxValue*: a {{float}} scalar. Specifies the maximum value of the range. When it is not specified, the clamping is not performed on the upper limit of the range.
+    **Returns:**
+        - an {{MLOperand}}. The output tensor of the same shape as *operand*.
+</div>
+
+<details open>
+  <summary>
+    The {{MLGraphBuilder/clamp(operand, options)}} method steps are:
+  </summary>
+  <div algorithm=clamp class=algorithm-steps>
+    1. Let |operand| be the first argument.
+    1. Let |options| be the second argument.
+        1. If running the <a>check clamp options</a> steps with |options| returns `false`, then throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+    1. If any of the following sub-steps fail, throw an "{{OperationError}}" {{DOMException}} and stop.
+        1. Let |output| be the result of invoking the <a>copy MLOperand</a> steps given |operand|.
+        1. Make a request to the underlying platform to:
+            1. Create an [=implementation-defined=] platform operator |clampImpl| for this method, given |options|.{{MLClampOptions/minValue}} and |options|.{{MLClampOptions/minValue}}.
+            1. Store a reference of |clampImpl| in |output|.{{MLOperand/[[operator]]}}.
+            1. Create an [=implementation-defined=] platform operand |outputImpl| to represent clamp output, given |output| and |clampImpl|.
+            1. Store a reference to |outputImpl| in |output|.{{MLOperand/[[operand]]}}.
+        1. Connect |operand|.{{MLOperand/[[operand]]}} as input to |clampImpl|.
+        1. Connect |output|.{{MLOperand/[[operand]]}} as output to |clampImpl|.
+    1. Return |output|.
+  </div>
+</details>
+
+#### The {{MLGraphBuilder/clamp(options)}} method #### {#api-mlgraphbuilder-clamp-options}
+<div>
+    **Arguments:**
+        - *options*: an optional {{MLClampOptions}}. The optional parameters of the operation.
+            - *minValue*: a {{float}} scalar. Specifies the minimum value of the range. When it is not specified, the clamping is not performed on the lower limit of the range.
+            - *maxValue*: a {{float}} scalar. Specifies the maximum value of the range. When it is not specified, the clamping is not performed on the upper limit of the range.
+    **Returns:**
+        - an {{MLActivation}}. The operator representing the clamp operation.
 </div>
 
+<details open>
+  <summary>
+    The {{MLGraphBuilder/clamp(options)}} method steps are:
+  </summary>
+  <div algorithm=clamp-options class=algorithm-steps>
+    1. Let |options| be the first argument.
+        1. If running the <a>check clamp options</a> steps with |options| returns `false`, then throw a "{{TypeError}}" {{DOMException}} and abort these steps.
+    1. Let |op| be the result of invoking the <a>create MLActivation</a> steps with `"clamp"` and |options|.
```

Comment:
what is `|op|`?

Reply:
It is a new variable named |op|, initialized to the result of invoking the said algorithm. Again, this is standard spec prose.